src/Tribe/Tickets_Handler.php

```diff
@@ -1185,6 +1183,21 @@ public function get_ticket_list_markup( $tickets = array() ) {
 	}
 
 	/**
+	 * Returns the markup for the tickets list in the tickets metabox
```

Comment:
Is this comment right? Sorry if I missed it first pass, but should it read "Returns the markup for the settings panel" or maybe "capacity table"?